oat-sa/extension-tao-itemqti
# Feat/aut 2545/prevent internal stylesheet from editor

**Related to:** [AUT-2545](https://oat-sa.atlassian.net/browse/AUT-2545)

**Description:**
Prevent issues with the **tao-user-styles**. 
Hide it on the stylesheet panel, and leave only the download action on the MediaManager modal.

**Companion PR**
- https://github.com/oat-sa/extension-tao-mediamanager/pull/421

**Changes:**

- Remove the **tao-user-styles** from the stylesheet panel editor.
- Fix the stylesheet download URL on the resource manager modal.
- Hide select and delete actions on the media resource modal for the **tao-user-styles** and show the download action.

**How to check:**

- Check the stylesheet panel to see if **tao-user-styles** is hidden.
- Add some custom styles on the panel
- Open the resource manager modal to add a new CSS file.
- Check the actions of the file on the media manager modal: 
      - All actions for the CSS stylesheet. 
      - Only download for **tao-user.styles**. 
- Download a CSS file from the resource manager modal and check if the file is ok.

Kitchen environment https://oat-sa.atlassian.net/browse/AUT-2545?focusedCommentId=199415
